calcom/cal.com
# feat: km-localization-cambodia 🇰🇭

## Added Localization for Khmer Language[Cambodai] 🇰🇭

i saw this project is use full for Cambodia people so i want to added my language to this projet



Fixes # (issue)


## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? ❌ 
- What are the minimal test data to have? ❌ 
- What is expected (happy path) to have (input and output)? - > add new langauage
- Any other important info that could help to test that PR 

## Mandatory Tasks


## Checklist
